migrations/20180701162442-create-product.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
Is this file need?
If no just delete this, pls

Reply:
Done.